topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md

```diff
@@ -35,21 +35,21 @@ contributors:
+Depending on the analyte of interest, the sample type and the mass spectrometer, the sample preparation steps as well as the properties of the acquired data vary. Human tissues are often stained with hematoxylin and hemalaun (H&E) after mass spectrometry imaging. This allows better visualization of tissue histology and can be used to compare the MSI heatmaps with the real morphology. The stained image of the tissue can also be used to define regions of interest that can be extracted from the MSI data and can be subjected to supervised statistical analysis. 
 
-As MSI is measuring only unfractionated molecules (MS1), identification of the resulting m/z features requires matching them to a list of known m/z features. The easiest approach is to match the MSI m/z features to existing databases such as uniprot for proteins or lipidmaps for lipids. More accurate identification is obtained by performing additional mass spectrometry experiments (either on the same or on a similar sample, e.g. adjacent slice from a tissue) that measure also the fragmented molecules (MS2) and thereby allows accurate identificaiton also of isobaric molecules. This tandem mass spectrometry approach can either be done in situ directly on the sample or by transferring the sample to a tube and bring it into solution for liquid-chromatography tandem mass spectrometry (LC-MS/MS). 
+As MSI measures only unfractionated molecules (MS1), identification of the resulting m/z features requires matching them to a list of known m/z features. The easiest approach is to match the MSI m/z features to existing databases such as UniProt for proteins or lipidmaps for lipids. More accurate identification is obtained by performing additional mass spectrometry experiments (either on the same or on a similar sample, e.g. adjacent slice from a tissue) that also measure the fragmented molecules (MS2) and thereby also allow accurate identification of isobaric molecules. This tandem mass spectrometry approach can either be done in situ directly on the sample or by transferring the sample to a tube and bringing it into solution for liquid-chromatography tandem mass spectrometry (LC-MS/MS). 
 
-In this tutorial we will determine and identify N-linked glycans by differential analysis of PNGase F treated and control murine kidney sections. The data analysis steps can be transferred to any other application that requires finding differential analytes such as for biomarker discovery. 
+In this tutorial we will determine and identify N-linked glycans by differential analysis of a control murine kidney sections with a section treated with PNGase F. The data analysis steps can be transferred to any other application that requires finding differential analytes, such as for biomarker discovery. 
 
```

Comment:
thanks a lot @simonbray for the corrections! Here I think it should be: of a control murine kidney section (without 's')?.

Reply:
Yes, thanks for catching that.